Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM nvidia/cuda:11.3.0-cudnn8-runtime-ubuntu18.04
+
+RUN \
+    apt-get update && \
+    apt-get -y install wget \
+    aria2 \
+    rsync \
+    cmake \
+    build-essential \
+    curl \
+    git \
+    vim \
+    nvidia-cuda-toolkit
+
+# install miniconda
+RUN \
+    curl https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -o miniconda.sh && \
+    sh miniconda.sh -b -p /tools/miniconda && \
+    rm miniconda.sh && \
+    /tools/miniconda/bin/conda init && \
+    /tools/miniconda/bin/conda clean -a
+ENV PATH "$PATH:/tools/miniconda/bin"
+
+COPY . /openfold
+WORKDIR /openfold
+
+# Create environment
+RUN conda env create -f environment.yaml
+SHELL ["conda", "run", "-n", "openfold-env", "/bin/bash", "-c"]
```

Comment:
the env name elsewhere in the repo is openfold_venv, might as well keep it consistent for consistency